awx/api/serializers.py

```diff
@@ -773,6 +774,16 @@ def get_summary_fields(self, obj):
                 if val is not None:
                     summary_fields['source_workflow_job'][field] = val
 
+        if self.is_detail_view:
+            ancestor = obj.ancestor_job
+            if ancestor != obj:
```

Comment:
Do we actually want to make this conditional?  I would think it would be nice for the UI code if they didn't have to defensively check if this was here and fall back to the full data of the current job if not.

Reply:
I think so (reasons privately discussed).